server/webhookNotifier.ts

```diff
@@ -250,14 +247,39 @@ const send = async (models, content: WebhookContent) => {
             },
           }]
         };
-      } else if (url.indexOf('matrix') !== -1) {
-        // TODO: matrix format and URL pattern matcher unimplemented
-        // return {
-        //   'text': `${getFiltered(content, address).join('\n')}`,
-        //   'format': 'plain',
-        //   'displayName': 'Commonwealth',
-        //   'avatarUrl': 'http://commonwealthLogoGoesHere'
-        // };
+      } else if (url.indexOf('telegram.org') !== -1) {
+        const getUpdatesUrl = url.split('/@').slice(0, -1).join('@');
+
+        var getChatUsername = url.split("/@");
+        getChatUsername = '@'+getChatUsername[1];
+
+        url = getUpdatesUrl+'/sendMessage'
+
+        webhookData = isChainEvent ? {
+          chat_id: getChatUsername,
+          text: `<a href="${chainEventLink}"><b>${title}</b></a>\n\n${fulltext}`,
+          parse_mode: 'HTML',
+          reply_markup: {
+            "resize_keyboard": true,
+            "inline_keyboard": [
+              [
+                {"text": "Read more on commonwealth", "url": chainEventLink}
+              ]
+            ]
+          }
+        } : {
+          chat_id: getChatUsername,
+          text: `<b>Author:</b> <a href="${actorAccountLink}">${actor}</a>\n<a href="${actedOnLink}"><b>${notificationTitlePrefix + actedOn}</b></a> \r\n\n${notificationExcerpt.replace(REGEX_EMOJI, '')}`,
+          parse_mode: 'HTML',
+          reply_markup: {
+            "resize_keyboard": true,
+            "inline_keyboard": [
+              [
+                {"text": "Commonwealth", "url": chainEventLink}
```

Comment:
Is this url accurate?

Reply:
Honestly, not sure. There's no way to test on-chain events that I know off so I just added the variables that was found in the on-chain format for discord.

Reply:
In the discord branch, they use a different url for non-chain events.

Reply:
I added a new link, tell me if it's correct

Reply:
Seems like the same link to me, in the inline_keyboard...